common/tools/dev-tool/src/commands/run/testNodeTsxTS.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license
+
+import concurrently from "concurrently";
+import { leafCommand, makeCommandInfo } from "../../framework/command";
+import { runTestsWithProxyTool } from "../../util/testUtils";
+import { createPrinter } from "../../util/printer";
+
+export const commandInfo = makeCommandInfo(
+  "test:node-tsx-ts",
+  "runs the node tests using mocha with the default and the provided options; starts the proxy-tool in record and playback modes",
+  {
+    "no-test-proxy": {
+      shortName: "ntp",
+      kind: "boolean",
+      default: false,
+      description: "whether to disable launching test-proxy",
+    },
+  },
+);
+
+export default leafCommand(commandInfo, async (options) => {
+  const reporterArgs =
+    "--reporter ../../../common/tools/mocha-multi-reporter.js --reporter-option output=test-results.xml";
+  const defaultMochaArgs = `${reporterArgs} --full-trace`;
+  const updatedArgs = options["--"]?.map((opt) =>
+    opt.includes("**") && !opt.startsWith("'") && !opt.startsWith('"') ? `"${opt}"` : opt,
+  );
+  const mochaArgs = updatedArgs?.length
+    ? updatedArgs.join(" ")
+    : '--timeout 1200000 --exclude "test/**/browser/*.spec.ts" "test/**/*.spec.ts"';
+  const command = {
+    command: `mocha --require tsx ${defaultMochaArgs} ${mochaArgs}`,
```

Comment:
Glad to see the `require` hook worked out 👍 still weird that `--import` was not working as expected on versions it was supposed to...

Reply:
Fixed with require for now